palindrome.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>    //This is Armstrong Park's program on determining whether a string of characters is a palindrome or not
+#include <string.h>
```

Comment:
Nice job, but you forgot a few test cases! 
1. the empty string - this is actually a palindrome, but your program doesn't handle it as such.
2. Capitals! Hannah should be returned as a palindrome, but isn't because of the capital H!